main.py

```diff
@@ -1,7 +1,55 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = ["George Klooney",
```

Comment:
You could keep the actor's name and surname in a tuple here so that you will use unpacking in a loop on line 19 as mentioned in the [checklist](https://github.com/mate-academy/py-genres-and-actors/blob/master/checklist.md).